ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.5-presubmits.yaml

```diff
@@ -103,7 +103,6 @@ presubmits:
     labels:
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-ovn-kubernetes-release-4.5-e2e-gcp-ovn
-    optional: true
```

Comment:
dropping `optional` is surprising to me.  @JacobTanenbaum , @squeed ,  can you speak to the indended behavior of e2e-gcp-ovn in 4.5?

Reply:
not sure what the best practice is; we also run e2e-aws, which is not optional.